slides/solidjs.md

```diff
@@ -1,7 +1,590 @@
+  });
+
+  return <div>pink</div>;
+}
+```
+
+
+[playground](https://playground.solidjs.com/?version=1.4.1#NobwRAdghgtgpmAXGGUCWEwBowBcCeADgsrgM4Ae2YZA9gK4BOAxiWGjIbY7gAQi9GcCABM4jXgF9eAM0a0YvADo1aAGzQiAtACsyAegDucAEYqA3EogcuPfr1oQAsgwi4sDiAGE1cKBHpCKVl5RRU6DW09CysrGXoIZlw0R15mRwg4JIAVWgARKFwoAGUGFjgACgBKfiteNMcIuAA6NVoAcwrw+hMyZkY0EzgRFSrLCHqhXCYJ6t4AXgA+e3SIJtaOrrAEsh6+gaGRsDGpK0lYiHjE5NSvBUI5kDrPFwTcCrml2on6zx8-AIPVaZHL5QolMqsapjZ6SGEQZ5TGa8AA8IjQADdFoQMABrFH6dFY8bnBGJRp8BIwVx8eaCYRiRgfGpfFF3Ti8fSLDwiWjMejwNzNdpwXAAUV8gtwACF8ABJERbKCEQijACE8KsZFF2Q4cAY7ypNI8AFYAAwWsZgSQAXSAA)
+
+---
+layout: two-cols
+---
+
+
+<header>
+<h1>Clean up after the component is unmounted</h1>
+</header>
+
+
+## React
+
+```ts
+import { useEffect } from 'react';
+
+function Comp() {
+   useEffect(() => {
+    return () => {
+      console.log("cleanup");
+    }
+   }, []); 
+
+   return <div>pink</div>;
+}
+```
+
+[playground](https://stackblitz.com/edit/react-ts-sibmkh?file=index.tsx)
+
+::right::
+
+## Solid
+
+```ts
+import { onCleanup } from "solid-js";
+
+function Comp() {
+  onCleanup(() => {
+    console.log("cleanup");
+  });
+
+  return <div>pink</div>;
+}
+```
+
+[playground](https://playground.solidjs.com/?version=1.4.1#NobwRAdghgtgpmAXGGUCWEwBowBcCeADgsrgM4Ae2YZA9gK4BOAxiWGjIbY7gAQi9GcCABM4jXgF9eAM0a0YvADo1aAGzQiAtACsyAegDucAEYqA3EogcuPfr1oQAwmrhQI9QlNnzFKuhraehZWVjL0EMy4aI68TgqEABQAlPxWvA7Oru6eiSm8ALwAfGkQGRnMjgFwAHRqtADmiSrM2R6EKsmWZVJdoT1CuExlADwiaABuRYQYANYj+uNT3ZL9lRBkfBEwDBC4AIKEXgWCwmKMeanFvCPxnLz6RVi8IrTM9PB7NQ1wuACirk+uAAQvgAJIiZpgKBHToAQj6ECsZF+ABUOHAGLhEttdgcjs8ACwABlJiLAkgAukA)
+
+---
+
+# Ecosystem | Libraries
+
+<div class="solidjs-click-list solidjs-gap-min">
+<v-clicks>
+
+- [react-use](https://github.com/streamich/react-use) <mdi-arrow-right /> [solid-primitives](https://github.com/solidjs-community/solid-primitives)
+- [react-router](https://reactrouter.com/en/main) <mdi-arrow-right /> [solid-router](https://github.com/solidjs/solid-router)
+- [react-transition-group](https://reactcommunity.org/react-transition-group/) <mdi-arrow-right />  [solid-transition-group](https://github.com/solidjs/solid-transition-group)
+- [react-testing-library](https://testing-library.com/docs/react-testing-library/intro/) <mdi-arrow-right /> [solid-testing-library](https://github.com/solidjs/solid-testing-library)
+- [@tanstack/react-query](https://tanstack.com/query/v4/docs/adapters/react-query) <mdi-arrow-right /> [@tanstack/solid-query](https://tanstack.com/query/v4/docs/adapters/solid-query)
+- [@tanstak/react-table](https://tanstack.com/table/v8/docs/adapters/react-table) <mdi-arrow-right /> [@tanstack/solid-table](https://tanstack.com/table/v8/docs/adapters/solid-table)
+- [react-window](https://react-window.vercel.app/#/examples/list/fixed-size) <mdi-arrow-right />  [@tanstak/solid-virtual](https://tanstack.com/virtual/v3/docs/adapters/solid-virtual)
+- [...and much more](https://www.npmjs.com/search?q=solidjs)
+
+</v-clicks>
```

Comment:
There is too much stuff here, but since you don't have to read them you could make this slide "lighter" by removing the `v-clicks` wrapper